packages/fluentui/react-northstar/src/utils/positioner/Popper.tsx

```diff
@@ -292,6 +293,35 @@ export const Popper: React.FunctionComponent<PopperProps> = props => {
           fn: handleUpdate,
         },
         popperInitialPositionFix.modifier,
+
+        autoSize && {
+          // Similar code as popper-maxsize-modifier: https://github.com/atomiks/popper.js/blob/master/src/modifiers/maxSize.js
+          // popper-maxsize-modifier only calculates the max sizes. This modifier applies the max sizes when overflow is detected
```

Comment:
Just wondering, should we submit a PR with this change?

Reply:
Sounds like a good idea. I'll make one and link this PR